src/WebJobs.Script/Config/ScriptHostOptionsSetup.cs

```diff
@@ -46,6 +46,17 @@ public void Configure(ScriptJobHostOptions options)
                 {
                     options.FileLoggingMode = fileLoggingMode.Value;
                 }
+
+                //var azureMonitorEnabled = jobHostSection.GetSection(ConfigurationSectionNames.Logging)
+                //    ?.GetValue<bool?>("azureMonitorEnabled");
+                //if (azureMonitorEnabled != null)
+                //{
+                //    options.AzureMonitorEnabled = azureMonitorEnabled.Value;
+                //}
+                //else
+                //{
+                //    options.AzureMonitorEnabled = true;
+                //}
```

Comment:
Did you mean to leave this commented out?

Reply:
Will remove - good catch